src/plugins.ts

```diff
@@ -9,26 +8,22 @@ import * as prettierParserHTML from 'prettier/plugins/html'
 import * as prettierParserMeriyah from 'prettier/plugins/meriyah'
 import * as prettierParserPostCSS from 'prettier/plugins/postcss'
 import * as prettierParserTypescript from 'prettier/plugins/typescript'
+import { loadIfExists, maybeResolve } from './resolve'
 
 interface PluginDetails {
   parsers: Record<string, Parser<any>>
   printers: Record<string, Printer<any>>
 }
 
 async function loadIfExistsESM(name: string): Promise<Plugin<any>> {
-  try {
-    if (req(import.meta.url).resolve(name)) {
-      let mod = await import(name)
-      return mod.default ?? mod
-    }
+  let mod = await loadIfExists<Plugin<any>>(name)
 
-    throw new Error('unreachable')
-  } catch (e) {
-    return {
-      parsers: {},
-      printers: {},
-    }
+  mod = mod ?? {
```

Comment:
```suggestion
  mod ??= {
```

Came out in Node 15, so might be too recent?

Reply:
esbuild's target is Node 14 so it should take care of this